cmd/podman/rmi.go

```diff
@@ -64,22 +64,15 @@ func rmiCmd(c *cli.Context) error {
 	}
 
 	for _, arg := range imagesToDelete {
-		image, err := runtime.GetImage(arg)
-		if err != nil {
-			if lastError != nil {
-				fmt.Fprintln(os.Stderr, lastError)
-			}
-			lastError = errors.Wrapf(err, "could not get image %q", arg)
-			continue
-		}
-		id, err := runtime.RemoveImage(image, c.Bool("force"))
+		image := runtime.NewImage(arg)
+		iid, err := image.Remove(c.Bool("force"))
 		if err != nil {
 			if lastError != nil {
 				fmt.Fprintln(os.Stderr, lastError)
 			}
 			lastError = errors.Wrapf(err, "failed to remove image")
```

Comment:
I am not sure we should wrap at this stage, since the error was already wrapped once.

I think 
lastError=err 

Would be fine.